mix.exs

```diff
@@ -27,19 +27,19 @@ defmodule BambooSmtp.Mixfile do
 
   defp deps do
     [
-      {:bamboo, "~> 1.2"},
-      {:credo, "~> 1.4.0", only: [:dev, :test]},
+      {:bamboo, "~> 1.6"},
+      {:credo, "~> 1.4.1", only: [:dev, :test]},
       {:earmark, ">= 1.3.2", only: :docs},
-      {:excoveralls, "~> 0.12.0", only: :test},
+      {:excoveralls, "~> 0.13.3", only: :test},
       {:ex_doc, ex_doc_version(), only: :docs},
-      {:gen_smtp, "~> 0.15.0"},
+      {:gen_smtp, "~> 1.0.1"},
```

Comment:
shouldn't we bump `bamboo_smtp` as well if we do this ?

Reply:
Yeah sure, should it be a minor version upgrade?

Reply:
actually not ! As long as we don't publish it's ok, but this is another PR I think